TS04-Dplyr-And-Time-Series-In-R.Rmd

```diff
@@ -45,94 +47,81 @@ After completing this lesson, you will:
-* [Convert Date & Time Data from Character Class to Date-Time Class (POSIX) in R]({{ site.baseurl}}R/Time-Series-Convert-Date-Time-Class-POSIX/)
-* [Refining Time Series Data - NoData Values and Subsetting Data by Date in R]({{ site.baseurl}}/R/Subset-Data-and-No-Data-Values/)
-* [Subset and Manipulate Time Series Data with dplyr]({{ site.baseurl}}/R/Time-Series-Subset-dplyr/)
-* [Plotting Time Series with ggplot in R]({{ site.baseurl}}/R/Time-Series-Plot-ggplot/)
-* [Plot uisng Facets and Plot Time Sereis with NDVI data]({{ site.baseurl}}/R/Time-Series-Plot-Facets-NDVI/)
-* [Converting to Julian Day]({{ site.baseurl}}/R/julian-day-conversion/)
+**Challenge Code:** NEON Data lesson often contain challenges that reinforce 
+learned skills. If available, the code for challenge solutions is found in a 
+downloadable `R` script available on the footer of each lesson page.
 
-###Sources of Additional Information
 
 </div>
 
-#Introduction to dplyr
-Working with micro-meteorology data from Harvard Forest, we are going to
-learn skills that will enable us to manipulating our data to group and summarize
-by year, daily average, and julian date.  To do this, we will use the `dplyr`
-package.
+****
+
+### Additional Resources
 
-The `dplyr` package is designed to simplify more complicated data manipulations
-in data frames.  `dplyr` is a powerful package that has capabilities beyond the 
-simple data manipulations we will cover in this lesson, including manipulating 
-data stored in an external database. This is especially useful to know about if 
-you will be working with very large datasets or relational databases in the 
-future. 
+* NEON Data Skills on <a href="http://neondataskills.org/R/GREPL-Filter-Piping-in-DPLYR-Using-R/" target="_blank"> spatial data and piping with dplyr</a> or 
+* Data Carpentry on <a href="http://www.datacarpentry.org/R-ecology/04-dplyr.html" target="_blank">Aggregating and Analyzing Data with dplyr</a> 
+* <a href="https://cran.r-project.org/web/packages/dplyr/dplyr.pdf" target="_blank"> DPLYR package description</a>.
+* <a href="http://blog.rstudio.org/2014/01/17/introducing-dplyr/" target="_blank">RStudio Introduction to DPLYR</a>
 
-If you are interested in learning more about `dplyr` consider following up with 
-`dplyr` lessons from 
+****
 
-1. NEON Data Skills on <a href="http://neondataskills.org/R/GREPL-Filter-Piping-in-DPLYR-Using-R/" target="_blank"> spatial data and piping with dplyr</a> or 
-2. Data Carpentry on <a href="http://www.datacarpentry.org/R-ecology/04-dplyr.html" target="_blank">Aggregating and Analyzing Data with dplyr</a> 
 
-and reading the CRAN `dplyr` <a href="https://cran.r-project.org/web/packages/dplyr/dplyr.pdf" target="_blank"> package description</a>.
+#Introduction to dplyr
+The `dplyr` package simplifies and increases efficiency of complicated yet commonly 
+performed data "munging" tasks. It uses the `data_frame` object as both an input and 
```

Comment:
it's used pretty widely in workshops, etc but it could confuse some:

Changed to:
"wrangling" (manipulation / processing) tasks.